### ISSUE TITLE
[A11Y][Navigation Button] Disabled state is not vocalized

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#931]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iOS and Android

### OS Version

iOS and Android

### App version

2.1.0 (718)

### Describe the issue

Disabled state is not vocalized by screenreaders (Android and iOS)

### Expected Behavior

Disabled state should be vocalized, like the button component

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (730)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 931
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (730)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 931
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility team)

**Comment by pya35** [comment]
fixed
